### PR TITLE
Added docs for evse_wifi integration

### PR DESCRIPTION
**Pull request by JFK344**
## Proposed change

Added the documentation for the new evse_wifi integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #
https://github.com/home-assistant/core/pull/85438
## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/evse_wifi.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ha_category:
+  - Car
+ha_release: 0.38
+ha_iot_class: Local Polling
+ha_domain: evse_wifi
+ha_platforms:
+  - sensor
+  - switch
+  - button
+  - number
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]: Initial additions / PRs to Home Assistant (including the actual integration), may only contain a single platform at the time. The initial documentation PR this cannot contain multiple platforms either.

File: source/_integrations/evse_wifi.markdown [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+title: EVSE WiFi
+description: Instructions on how to integrate a WiFi-equipped Simple EVSE Charging station with Home Assistant
+ha_category:
+  - Car
+ha_release: 0.38
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]: Unlikely 😉

File: source/_integrations/evse_wifi.markdown [hunk 3]
```diff
@@ -0,0 +1,49 @@
+---
+title: EVSE WiFi
+description: Instructions on how to integrate a WiFi-equipped Simple EVSE Charging station with Home Assistant
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]:
```suggestion
description: Instructions on how to integrate a Wi-Fi-equipped Simple EVSE Charging station with Home Assistant
```

File: source/_integrations/evse_wifi.markdown [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - button
+  - number
+ha_integration_type: integration
+---
+
+This EVSE WiFi device platform pulls data from an [SimpleEvse-WiFi](https://github.com/CurtRod/SimpleEVSE-WiFi) Charging station equipped with an ESP8266-based Wi-Fi connection.
+
+## Configuration
+
+To enable this sensor in your installation, search for "Evse-Wifi" in the Integrations and add it via the Configuration-Flow
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]:
```suggestion
{% include integrations/config-flow.md %}
```

File: source/_integrations/evse_wifi.markdown [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+- Reboot Evse ESP
+
+## Switches 
+
+- Charge State
+
+## Numbers
+
+- Charge Current
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]: Please limit this PR to a single platform.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/evse_wifi.markdown [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+This EVSE WiFi device platform pulls data from an [SimpleEvse-WiFi](https://github.com/CurtRod/SimpleEVSE-WiFi) Charging station equipped with an ESP8266-based Wi-Fi connection.
+
+## Configuration
+
+To enable this sensor in your installation, search for "Evse-Wifi" in the Integrations and add it via the Configuration-Flow
```
Comment (frenck) [inline source/_integrations/evse_wifi.markdown]:
```suggestion
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by JFK344** [comment]
Yes 🥳  thank you 👍 now lets see how the acutal integration 😅

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**